### PR TITLE
CP-2919 [API Device][Multidevice] Device drawer - stress test

### PR DESCRIPTION
**Pull request by robertmudi**
JIRA Reference: [CP-2919]

### :memo: Description ️

Open App
Connect 1st Kompakt
Connect 2nd Kompakt
Wait until Device drawer shows up 
Verify Device Drawer devices
Connect 3rd Kompakt
Verify Device Drawer devices
Connect 4th Kompakt, Connect 5th Kompakt, Connect 6th Kompakt
Verify Device Drawer devices (Names and serial numbers) and UI and scroll
Click 6th Kompakt 
Verify Overview - and a new Serial Number
Click Show Connected Devices
Disconnect all Kompakts
Verify Welcome Page Screen is displayed

[CP-2919]: https://appnroll.atlassian.net/browse/CP-2919?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    E2EMockClient.addDevice({
+      path: "path-3",
+      serialNumber: thirdSerialNumber,
+    })
+
+    await browser.pause(6000)
+    const menuItem = await $(`//a[@href="#/generic/mc-overview"]`)
+
+    await menuItem.waitForDisplayed({ timeout: 10000 })
+    await expect(menuItem).toBeDisplayed()
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts:114]: 🔴  `mockdev.51` 

```
Error: Expect $(`[data-testid="device-select-drawer-content"]`) to be displayed

Expected: "displayed"
Received: "not displayed"
```

File: apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    const fourthDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      fourthSerialNumber
+    )
+    const fifthDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      fifthSerialNumber
+    )
+    const sixthDeviceOnDrawer = await drawerPage.getDeviceOnDrawer(
+      sixthSerialNumber
+    )
+    await expect(fourthDeviceOnDrawer).toBeDisplayed()
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts:188]: 🔴 mockdev.51
```
Error: Expect $(`[data-testid="device-select-drawer-content"]`) to be displayed

Expected: "displayed"
Received: "not displayed"
```

File: apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+  it("Disconnect the devices and check if Welcome Page is present", async () => {
+    E2EMockClient.removeDevice("path-1")
+    E2EMockClient.removeDevice("path-2")
+    E2EMockClient.removeDevice("path-3")
+    E2EMockClient.removeDevice("path-4")
+    E2EMockClient.removeDevice("path-5")
+    E2EMockClient.removeDevice("path-6")
+
+    const homeHeader = await HomePage.homeHeader
+    await homeHeader.waitForDisplayed()
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts:243]: 🔴  mockdev.51
```
[0-0] Error in "Kompakt switching devices.Disconnect the devices and check if Welcome Page is present"
Error: element ("h1[data-testid='onboarding-title-header']") still not displayed after 6000ms
```

**Review by tomaszmaleckimudita — CHANGES_REQUESTED** [review]

File: apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts (hunk 1 above)
Comment (robertmudi) [reply apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts:114]: [chrome 116.0.5845.228 windows #0-0]
[chrome 116.0.5845.228 windows #0-0] » \src\specs\stress-tests\device-drawer-stress-test.ts
[chrome 116.0.5845.228 windows #0-0] Kompakt switching devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 1,2 Kompakt devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Devices 1 and 2 on Drawer and check their Serial Numbers
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 3rd Kompakt device
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Device 3 on the Drawer and check the Serial Number
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 4,5,6 Kompakt devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Devices 4,5 and 6 on Drawer and check their Serial Numbers, scroll down and select 6th Kompakt
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify 6th Kompakt with new Serial Number and click Show connected devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Disconnect the devices and check if Welcome Page is present
[chrome 116.0.5845.228 windows #0-0]
[chrome 116.0.5845.228 windows #0-0] 8 passing (42.7s)


Spec Files:      1 passed, 1 total (100% completed) in 00:00:45

mockdev 51

File: apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts (hunk 3 above)
Comment (robertmudi) [reply apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts:243]: [chrome 116.0.5845.228 windows #0-0]
[chrome 116.0.5845.228 windows #0-0] » \src\specs\stress-tests\device-drawer-stress-test.ts
[chrome 116.0.5845.228 windows #0-0] Kompakt switching devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 1,2 Kompakt devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Devices 1 and 2 on Drawer and check their Serial Numbers
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 3rd Kompakt device
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Device 3 on the Drawer and check the Serial Number
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 4,5,6 Kompakt devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Devices 4,5 and 6 on Drawer and check their Serial Numbers, scroll down and select 6th Kompakt
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify 6th Kompakt with new Serial Number and click Show connected devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Disconnect the devices and check if Welcome Page is present
[chrome 116.0.5845.228 windows #0-0]
[chrome 116.0.5845.228 windows #0-0] 8 passing (42.7s)


Spec Files:      1 passed, 1 total (100% completed) in 00:00:45

mockdev 51

File: apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts (hunk 2 above)
Comment (robertmudi) [reply apps/mudita-center-e2e/src/specs/stress-tests/device-drawer-stress-test.ts:188]: [chrome 116.0.5845.228 windows #0-0]
[chrome 116.0.5845.228 windows #0-0] » \src\specs\stress-tests\device-drawer-stress-test.ts
[chrome 116.0.5845.228 windows #0-0] Kompakt switching devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 1,2 Kompakt devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Devices 1 and 2 on Drawer and check their Serial Numbers
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 3rd Kompakt device
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Device 3 on the Drawer and check the Serial Number
[chrome 116.0.5845.228 windows #0-0]    ✓ Connect 4,5,6 Kompakt devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify Devices 4,5 and 6 on Drawer and check their Serial Numbers, scroll down and select 6th Kompakt
[chrome 116.0.5845.228 windows #0-0]    ✓ Verify 6th Kompakt with new Serial Number and click Show connected devices
[chrome 116.0.5845.228 windows #0-0]    ✓ Disconnect the devices and check if Welcome Page is present
[chrome 116.0.5845.228 windows #0-0]
[chrome 116.0.5845.228 windows #0-0] 8 passing (42.7s)


Spec Files:      1 passed, 1 total (100% completed) in 00:00:45

mockdev 51 ok

File: apps/mudita-center-e2e/wdio.conf.ts [hunk 4]
```diff
@@ -73,6 +73,7 @@ export const config: Options.Testrunner = {
     toRelativePath(TestFilesPaths.mcHomePageForceUpdateTest),
     toRelativePath(TestFilesPaths.kompaktAbout),
     toRelativePath(TestFilesPaths.kompaktConnectedDevicesModalStressTest),
+    toRelativePath(TestFilesPaths.kompakrDrawerStressTest),
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/wdio.conf.ts:76]:
```suggestion
    toRelativePath(TestFilesPaths.kompaktDrawerStressTest),
```

File: apps/mudita-center-e2e/wdio.conf.ts [hunk 5]
```diff
@@ -96,6 +97,7 @@ export const config: Options.Testrunner = {
       toRelativePath(TestFilesPaths.kompaktSwitchingDevices),
       toRelativePath(TestFilesPaths.kompaktAbout),
       toRelativePath(TestFilesPaths.kompaktConnectedDevicesModalStressTest),
+      toRelativePath(TestFilesPaths.kompakrDrawerStressTest),
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/wdio.conf.ts:100]:
```suggestion
    toRelativePath(TestFilesPaths.kompaktDrawerStressTest),
```

File: apps/mudita-center-e2e/wdio.conf.ts [hunk 6]
```diff
@@ -124,6 +126,7 @@ export const config: Options.Testrunner = {
...
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/wdio.conf.ts:129]:
```suggestion
    toRelativePath(TestFilesPaths.kompaktDrawerStressTest),
```

**Review by tomaszmaleckimudita — CHANGES_REQUESTED** [review]
You didn't commit `/mudita-center-e2e/src/test-filenames/consts/test-filenames.const.ts`

File: apps/mudita-center-e2e/wdio.conf.ts (hunk 4 above)
Comment (robertmudi) [reply apps/mudita-center-e2e/wdio.conf.ts]: fixed typo

File: apps/mudita-center-e2e/wdio.conf.ts (hunk 5 above)
Comment (robertmudi) [reply apps/mudita-center-e2e/wdio.conf.ts]: fixed typo

File: apps/mudita-center-e2e/wdio.conf.ts (hunk 6 above)
Comment (robertmudi) [reply apps/mudita-center-e2e/wdio.conf.ts]: fixed typo

**Comment by robertmudi** [comment]
You didn't commit /mudita-center-e2e/src/test-filenames/consts/test-filenames.const.ts

the file was commited byt there was a typo in kompakt name there, therefore all three typos in wdio.conf.ts were not observed by me, as I copy pasted them all

**Review by tomaszmaleckimudita — APPROVED** [review]

**Pull request merged**